tools/check_format.sh

```diff
@@ -8,7 +8,7 @@ TO_CHECK="${2:-$PWD}"
 bazel run @envoy//tools:code_format/check_format.py -- \
   --skip_envoy_build_rule_check  --namespace_check Nighthawk \
   --build_fixer_check_excluded_paths=$(realpath ".") \
-  --include_dir_order envoy,nighthawk,external/source/envoy,external,api,common,source,exe,server,client,test_common,test \
+  --include_dir_order envoy,nighthawk,external/source/envoy,external,api,common,source,exe,server,client,grpcpp,test_common,test \
```

Comment:
This `grpcpp` section is a simple way to prevent the script from rearranging the grpcpp, gtest, gmock headers into an order that is then considered invalid.

Reply:
Oh, that might have been a hassle to figure out, sorry about that. I wonder if we can improve there, ideally the "fix" doesn't produce something that the "check" objects to :-(

Reply:
This is definitely a quick fix. It seems to pass all the CI checks. I don't know how it's arriving at the order or how it's checking it. If we figure out how it works, this would no longer be necessary.

Reply:
I think it's the right tweak to address this. But I do find it's disappointing that check and fix don't line up when the order isn't explicitly specified. I'll file an issue to see if Envoy has progressed here, and if there are changes worth including in our version of this.